recipes/TheanoLM/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "TheanoLM" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "ebb312c34ad0bfea950f412f4657fe9ae6dc8ed8e904bb199ceef03faf50ff80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py<34]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
It's not like you _don't_ need `python` at runtime...I think conda uses python specifically for some things, should probably add that back in. Looks like the `setup.py` also explicitly requires `scipy` and `numpy`, so should probably include those here too.

Reply:
Ok, I'll add those three back in. It are also dependencies of Theano, so I thought to keep the recipe lean. On the other hand, I think at least numpy, but probably also scipy, is used directly in the package, so the direct dependency should be there.

Reply:
scipy was used only in the word class tool, which is included in the package, but is not used. It's not used because there are more efficient implementations. We could consider removing it from the package. At least we could remove the scipy dependency. numpy is used allover the package.